content/pages/about.md

```diff
@@ -2,7 +2,113 @@
 title: About Us
 path: "/about"
 tags: ["About"]
+excerpt: Sample courses page.
 ---
 
+<style>
```

Comment:
Why this page does not follow the React style? It has a css and html code hard-coded in a .md file. I suggest you create a little react component that has an image, a name and a job description and then you use this component to create the about us page. The same for the map.
Thank you for the effort.